nerfstudio/models/base_model.py

```diff
@@ -103,13 +106,22 @@ def populate_modules(self):
                 near_plane=self.config.collider_params["near_plane"], far_plane=self.config.collider_params["far_plane"]
             )
 
-    @abstractmethod
+        self.temporal_distortion = None
+        if getattr(self.config, "enable_temporal_distortion", False):
+            params = self.config.temporal_distortion_params
+            kind = params.pop("kind")
+            self.temporal_distortion = kind.to_temporal_distortion(params)
+
     def get_param_groups(self) -> Dict[str, List[Parameter]]:
         """Obtain the parameter groups for the optimizers
 
         Returns:
             Mapping of different parameter groups
         """
+        param_groups = {}
+        if self.temporal_distortion is not None:
+            param_groups["temporal_distortion"] = list(self.temporal_distortion.parameters())
```

Comment:
For the same reason as the configs, I don't think we want to include this distortion logic in the base model. If someone added temporal distortions in their model that were on by default, thus not needed a flag like `enable_temporal_distortion` this code would fail.